### PR TITLE
Fix #67

### LINKED ISSUE #67: Please tag a new stable release
The current stable release version `1.0.9` fails with this error message:
```console
$ cargo install cargo-sort
…
$ cargo sort -w           
Checking rust...
Finished: Cargo.toml for "rust" has been rewritten
Checking mycrate...
thread 'main' panicked at /Users/shaun.jackman/.cargo/registry/src/index.crates.io-6f17d22bba15001f/cargo-sort-1.0.9/src/sort.rs:113:46:
called `Result::unwrap()` on an `Err` value: TomlError { message: "TOML parse error at line 13, column 6\n   |\n13 | serde.workspace = true\n   |      ^\nUnexpected `.`\nExpected `=`\n" }
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```
and this issue is fixed on the current `main` branch. Please tag a new stable release. Thank you!

The workaround is to install `cargo-sort` from Git:
```sh
cargo install --git=https://github.com/DevinR528/cargo-sort --branch=main cargo-sort
```

### PR DESCRIPTION
**Pull request by ssrlive**

### CODE REVIEW

**Comment by jplatte** [comment]
I don't understand. What is being fixed here?

**Comment by ssrlive** [comment]
The comma you have been waiting for for a long time has appeared again.

**Comment by ssrlive** [comment]
So, are you retaliating against me by not moving forward? @jplatte

**Comment by jplatte** [comment]
No, I just haven't gotten around to trying this, because this is not a very high priority project for me but I want to test it because the PR does not include a test and it's not obvious to me how it would fix the formatting issue.

**Comment by ssrlive** [comment]
I think this commit can answer your question.

https://github.com/DevinR528/cargo-sort/pull/82/commits/4507082eca46b29680683b24de627253c3b7ec18

File: src/fmt.rs [hunk 1]
```diff
@@ -98,23 +106,17 @@ impl Config {
...
+            table_order: DEF_TABLE_ORDER.iter().map(|s| (*s).to_owned()).collect(),
         }
     }
 }
 
 impl FromStr for Config {
     type Err = &'static str;
     fn from_str(s: &str) -> Result<Self, Self::Err> {
+        if s.is_empty() {
+            return Ok(Config::default());
```
Comment (ssrlive) [inline src/fmt.rs:118]: Returning early here can slightly improve performance.

**Comment by ssrlive** [comment]
Hi @jplatte and @DevinR528 ,  if you really have no time or interest in maintaining this project, you can add me as a co-contributor and publisher. I'm interested in maintaining it.

**Comment by jplatte** [comment]
Please show some patience. It's not uncommon for people to need a couple of weeks to get to PRs to the open source projects  they maintain on the side. It's also the holiday season, and there's a lot of shit going on in my offline life (finishing house renovations before moving early next year).

**Comment by jplatte** [comment]
I incorporated the second commit as 3701753667c5d47ae166d6e5cd7a11384e068ed0, and the optimization from the first commit as 8063b2fef5e4a0f6779d3c0213b4c8aac80b7f47. I'm not clear on what the rest of the first commit is meant to achieve, or what issues the third commit from this PR fixes.

Separately, `cargo run` within the `cargo-sort` repository still collapses lines 33-35 (a few lines with comments), which I would also like to see fixed before making a release.

**Comment by ssrlive** [comment]
I think this is a bug in `toml-edit`, which always tries to merge multiple adjacent lines of separate comments into one line, no matter how long the new line is.

This may be a problem that `cargo-sort` cannot solve.

**Comment by ssrlive** [comment]
I think this is a bug in `toml-edit`, which always tries to merge multiple adjacent lines of separate comments into one line, no matter how long the new line is.

This may be a problem that `cargo-sort` cannot solve.

> Separately, cargo run within the cargo-sort repository still collapses lines 33-35 (a few lines with comments), which I would also like to see fixed before making a release.

**Pull request closed without merging**